docs/config_search.md

```diff
@@ -18,84 +18,132 @@ limitations under the License.
-[`concurrency`](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/perf_analyzer.md#request-concurrency)
-values 1 through 1024 increased exponentially (i.e. 1, 2, 4, 8, ...). The maximum value can be configured
-using the `run_config_search_max_concurrency` key in the Model Analyzer Config.
+---
 
-An example config that limits the search space used by Model Analyzer is
-described below:
+### [Request Concurrency Search Space](https://github.com/triton-inference-server/server/blob/master/docs/user_guide/perf_analyzer.md#request-concurrency)
+
+- `Default:` 1 to 1024 concurrencies, sweeping over powers of 2 (i.e. 1, 2, 4, 8, ...)
+- `--run-config-search-min-concurrency: <val>`: Changes the request concurrency minimum automatic search space value
+- `--run-config-search-max-concurrency: <val>`: Changes the request concurrency maximum automatic search space value
+
+---
+
+_An example YAML config that limits the search space:_
 
 ```yaml
 model_repository: /path/to/model/repository/
 
 run_config_search_max_instance_count: 3
-run_config_search_max_concurrency: 8
+run_config_search_min_model_batch_size: 16
+run_config_search_min_concurrency: 8
+run_config_search_max_concurrency: 256
 profile_models:
-  - model_1
-  - model_2
+  - model_A
 ```
 
-If any `model_config_parameters` are specified for a model, it will disable
-automatic searching of model configs and will only search within the values specified.
-If `concurrency` is specified then only those values will be tried instead of the default concurrency sweep.
-If both `concurrency` and `model_config_parameters` are specified, automatic
-config search will be disabled.
+_This will perform an Automatic Brute Search with instance counts: 3-5, batch sizes: 16-128, and concurrencies: 8-256_
 
-For example, the config specified below will only automatically sweep through
-the `model_config_parameters` that was described above:
+---
 
-```yaml
-model_repository: /path/to/model/repository/
+### **Interaction with Remote Triton Launch Mode**
 
-profile_models:
-  model_1:
-    parameters:
-      concurrency: 1,2,3,128
-```
+When the triton launch mode is remote, _\*\*only concurrency values can be swept._\*\*<br>
+
+Model Analyzer will ignore any model config parameters because we have no way of accessing and modifying the model repository of the remote Triton Server.
+
+---
+
+## Manual Brute Search
+
+**Default brute search mode when any model config parameters or parameters are specified**
+
+Using Manual Brute Search, you can create custom sweeps for any parameter that can be specified in the model configuration. There are two ways this mode is enabled when doing a brute search:
+
+- Any `model config parameters` are specified:
+  - You must manually specify the batch sizes and instance counts you want MA to sweep
+  - Request concurrencies will still be automatically swept (as they are not a model config parameter)
+- Any `parameters` are specified:
```

Comment:
What does parameters mean in this case?

Reply:
These are two separate sections in the config YAML: `model config parameters` (for instance and batch size) and `parameters` (where concurrency is set).

Reply:
Yeah I just don't think this is clear for the user. Do we have other doc sections that can be linked to give the user context of what those are and how they are specified?

Reply:
I don't think it's very clear either. You really need to be familiar in the art of writing a protobuf to understand the difference. We have extensive examples of `model config parameters` and `parameters` in `config.md`.

I think it's beyond the scope of our document to explain this to the user - my feeling is that they're either familiar with config files and they are going to understand this, or they aren't and will not care about this section as they won't be trying to run this niche case. 

Personally, I hate that we have this schism between automatic and manual, but this is how our logic works and I thought it was important to call it out to the user.

Reply:
Yeah but can't we link the existing docs for those parameter sections? For example, make `parameters` a clickable link to https://github.com/triton-inference-server/model_analyzer/blob/main/docs/config.md#test-configuration-parameter ?

Reply:
Done